gym_extensions/continuous/box2d/falcon_drone_ship_lander.py

```diff
@@ -25,7 +25,7 @@
 MAIN_ENGINE_POWER  = 100.0 # 13.0
 SIDE_ENGINE_POWER  =  3.0 # 0.6
 
-INITIAL_RANDOM = 500 #300.0   # Set 1500 to make game harder
+INITIAL_RANDOM = 500 # Set 1500 to make game harder
 
 FALCON_POLY =[
```

Comment:
Btw, i played around with it quickly and something like:


FALCON_POLY =[
    (-4,+150), (-13,135),(-13,0), (-13,-30),
    (+13,-30),(+13,0),(+13,+135), (+4,+150)
    ]

Looks more falcon-y in my opinion (also moving the side engines up to 140.0), but idk if you think this is necessary.